### PR TITLE
setting changed popup

### PR DESCRIPTION
**Pull request by renzocarara**
I added some code for the popup.
I implemented the 2 different solutions we talked about, for different screen sizes.

About the responsiveness,
I had some difficulties to understand the expected behavior for smaller size devices, when width or height is less than 550-600px,
It is not clear to me what is the expected behaviour.

Playing with width/height sizes,
I realized that also the tooltips, recently added, were not visible on smaller height device, they lay out of the viewport...
I put some small changes in stylesheet.css to resize the main window and give room for the tootltip, but I think more work is needed in general about responsiveness.

### CODE REVIEW

File: include/stylesheet.css [hunk 1]
```diff
@@ -43,11 +44,18 @@ body {
     z-index: 0;
 }
 
-@media ((max-width: 500px) or (max-height: 500px)) {
+/* @media ((max-width: 500px) or (max-height: 500px)) {
     #main_display {
         max-width: 97vh;
         max-height: 97vh;
     }
+} */
```
Comment (codencoding) [inline include/stylesheet.css:52]: Remove lines 47-52 in favor of lines 54-59

File: include/stylesheet.css [hunk 2]
```diff
@@ -43,11 +44,18 @@ body {
...
+
+@media screen and (max-width: 600px), (max-height: 600px) {
```
Comment (codencoding) [inline include/stylesheet.css:54]: Keep max-dimensions at 500px

File: include/stylesheet.css [hunk 3]
```diff
@@ -43,11 +44,18 @@ body {
...
+    #main_display {
+        width: 90vh;
+        height: 86vh;
+    }
```
Comment (codencoding) [inline include/stylesheet.css:58]: Keep using 97vh for these dimensions

**Review by codencoding — CHANGES_REQUESTED** [review]
Very solid work with the error notification, looks very smooth as functions exactly as intended.

One note though; until the desired responsiveness behavior is sorted out, lets keep the size of the #main_display at 97vh. Looking at the page on a landscape oriented mobile-device really pushes the #main_display out of the window and gives unintended effects. 

I'm thinking that instead of trying to accommodate the tooltips for smaller windows, the #main_display window can instead fill almost the entire screen instead. So instead of the @media #main_display max-dimensions using 97vh, they can use 97% instead. I don't like the idea of wasting a lot of space on top just for tooltips, so maybe just don't use them once the screen gets too small. Either that or re-work the tabs to have labels on them once the screen gets small enough for that (which sounds difficult given how large the tabs will have to be for the labels to be legible). Possibly have a text element somewhere near the header that states what the current tab is would be an easily implementable solution.

At this point please revise this pull request to only include the "setting changed" notifications and not modify the responsiveness of the #main_display window.

**Comment by renzocarara** [comment]
ok Chase,
I think I made all the requested changes about sizes. I restored the original values.
(I also inserted the popup notification on the "Change weather API key" button, it was missing.)

**Review by codencoding — APPROVED** [review]

**Pull request merged**